php7/scripts/install-composer.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
```

Comment:
Dit kan je tegenwoordig helemaal vervangen door:  
`COPY --from=composer:latest /usr/bin/composer /usr/bin/composer`

Multistage build is dat: https://docs.docker.com/develop/develop-images/multistage-build/

Reply:
Yeah, ik gebruik zelf al best veel multistage builds maar eigenlijk vooral om de eind-image zo klein mogelijk te houden. Composer zo installeren is een goed idee!